assets/scripts/ajaxModalSearch.js

```diff
@@ -0,0 +1,45 @@
+const searchForm = document.getElementById('search-form');
+const searchResult = document.getElementById('search-result');
+const url = searchForm?.getAttribute('data-ajax-url');
```

Comment:
No, this is not how you get data atributes : https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
now i know how to use the data attributes